### PR TITLE
Update name to Tools for MicroProfile

### LINKED ISSUE #23: Rename the extension as "Tools for MicroProfile"
See this communication from the Eclipse Management Organisation (EMO):

> [...] the name of the extension violates the Eclipse Foundation's Trademark Usage Guidelines.
> 
> MicroProfile is a registered trademark of the Eclipse Foundation. As such, it subject to the terms of the Eclipse Foundation Trademark Usage Guidelines.
> 
> Trademark management is mostly about avoiding confusion. In that regard, you cannot call a product "MicroProfile Foo" as this very tightly associates the brand with your product. Rather, you may use the form "Foo for MicroProfile". So, "MicroProfile Tools" is not acceptable, but "Tools for MicroProfile" is (having an actual name would be even better, but is not required to conform to our guidelines).
> 
> For completeness, you can use the MicroProfile logo, but you may not modify it or incorporate it into your product's logo.
> 
> You must also attribute the trademarks to the Eclipse Foundation by adding a statement along the lines of "MicroProfile(R) and the MicroProfile logo are trademarks of the Eclipse Foundation".

We'll release 0.1.1 with those corrections ASAP.

cc @waynebeaton

### PR DESCRIPTION
**Pull request by rzgry**
Fixes #23 

Signed-off-by: Ryan Zegray <ryan.zegray@ibm.com>

### CODE REVIEW

**Review by datho7561 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -162,3 +162,6 @@ Please report bugs, issues and feature requests by creating a [GitHub Issue](htt
 
 Apache License 2.0.
 See [LICENSE](LICENSE) file.
+
+
+MicroProfile(R) and the MicroProfile logo are trademarks of the Eclipse Foundation
```
Comment (fbricon) [inline README.md:167]: use ®

File: README.md [hunk 2]
```diff
@@ -76,9 +76,9 @@ To avoid having this error, you must configure the following in `settings.json`:
 This settings is set by default.
 
 
-## Extending MicroProfile Tools for VSCode
+## Extending Tools for MicroProfile
 
-By default MicroProfile Tools for VSCode provides:
+By default Tools for MicroProfile provides:
```
Comment (fbricon) [inline README.md:81]: By default,

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,7 @@
-# MicroProfile Tools for Visual Studio Code Changelog
+# Tools for MicroProfile Changelog
+
+## [0.1.1] (September 23, 2020)
+* Update name to Tools for MicroProfile. See [#23](https://github.com/redhat-developer/vscode-microprofile/issues/23)
```
Comment (fbricon) [inline CHANGELOG.md:4]: "Tools for MicroProfile"

File: README.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 [![Marketplace Version](https://vsmarketplacebadge.apphb.com/version/redhat.vscode-microprofile.svg "Current Release")](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-microprofile)
 [![Gitter chat for vscode-microprofile](https://badges.gitter.im/redhat-developer/vscode-microprofile.svg)](https://gitter.im/redhat-developer/vscode-microprofile)
 
-# MicroProfile Tools for Visual Studio Code
+# Tools for MicroProfile
 
 ## Description
```
Comment (fbricon) [inline README.md:8]: This Visual Studio Code extension provides support for the development of [MicroProfile®](https://microprofile.io/)-based applications, via:

**Review by fbricon — CHANGES_REQUESTED** [review]

**Comment by fbricon** [comment]
Thanks @rzgry

**Pull request merged**